docs/contributing/style-guides.md

```diff
@@ -1,7 +1,27 @@
 # Style guides
 
-TODO: Add some guidelines here.
```

Comment:
Maybe we may need to add a reference to the use of the `tailwind.config.cjs` and the other stuff you added in the other PR #228 

https://github.com/OpenVAA/voting-advice-application/blob/497a876af75108f9aace7d96871645167442e46b/frontend/README.md#tailwind-classes

Reply:
I added the part of that file in a new styling doc.